app/ui.js

```diff
@@ -139,9 +139,10 @@ var UI;
                 UI.toggleControlbarSide();
             }
 
-            // Setup and initialize event handlers
+            UI.initFullscreen();
```

Comment:
It isn't really related to setting up event handlers, with the new separation it felt to have more in common with the init functions